crates/nu-protocol/src/engine/engine_state.rs

```diff
@@ -100,8 +100,8 @@ pub struct EngineState {
     config_path: HashMap<String, PathBuf>,
     pub history_enabled: bool,
     pub history_session_id: i64,
-    // If Nushell was started, e.g., with `nu spam.nu`, the file's parent is stored here
-    pub(super) currently_parsed_cwd: Option<PathBuf>,
+    // Path to the script Nushell is currently running, if any.
+    pub script: Option<PathBuf>,
```

Comment:
Is this only used for scripts or does it have uses for other kinds of files (modules or config as in #12338 ) else the renaming is probably bad as we still attach some specific semantics in evaluation to scripts.

Reply:
I've renamed it to simply "file". It's less descriptive, but with the comments I've added, it should be fine?